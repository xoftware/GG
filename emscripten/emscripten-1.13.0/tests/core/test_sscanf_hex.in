#include "stdio.h"

int main() {
  unsigned int a, b;
  sscanf("0x12AB 12AB", "%x %x", &a, &b);
  printf("%d %d\n", a, b);
}
