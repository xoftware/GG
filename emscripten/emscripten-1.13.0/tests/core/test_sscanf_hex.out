4779 4779